streamingcli/jupyter/notebook_converter.py

```diff
@@ -159,6 +159,9 @@ def _handle_magic_cell(
             if NotebookConverter._skip_statement(sql_statement):
                 return []
             return self._convert_sql_statement_to_python_instructions(sql_statement)
+        if source.startswith("%%flink_execute"):
+            code = "\n".join(source.split("\n")[1:])
+            return [Code(value=code)]
         if source.startswith("%flink_execute_sql_file"):
```

Comment:
I don't think this line will be invoked anytime after adding few lines of code above :)